pom.xml

```diff
@@ -56,6 +56,7 @@
             <licenseName>epl_only_v2</licenseName>
             <failOnMissingHeader>true</failOnMissingHeader>
             <failOnNotUptodateHeader>true</failOnNotUptodateHeader>
+            <generateBundle>true</generateBundle>
```

Comment:
Sure this is needed? I think enabling `update-project-license` should add LICENCE.txt to the root of the JAR anyway.

Reply:
You seem to be right. :-)